src/utils/constants.js

```diff
@@ -124,6 +124,8 @@ export const ALERT_MESSAGE = {
 
 export const ALERT_MESSAGE_DURATION = 3000;
 
+export const USERS_NOT_OWNED = 'Players who do not have';
```

Comment:
Better name might be NOT_OWNED_BY_USERS

Reply:
Alternatively, USERS_WITHOUT, USERS_WITHOUT_MOUNT

Reply:
Renaming to `NOT_OWNED_BY_USERS`